recharges/tasks.py

```diff
@@ -108,6 +108,38 @@ class Hotsocket_Get_Airtime(Task):
     """
     name = "recharges.tasks.hotsocket_get_airtime"
 
+    def normalize_msisdn(self, msisdn, country_code='',):
```

Comment:
These two functions are lacking comments. Also, I think they're useful enough that we should keep them outside the class for reuse.